app/controllers/events_controller.rb

```diff
@@ -99,4 +102,12 @@ def authorize_user_manage
       redirect_to(root_path)
     end
   end
+
+  def get_role
+    @roles = {}
+    EventStaff.roles.each do |key, value|
+       role_text = key.split("_").map(&:capitalize).join(" ")
+       @roles[role_text] = key
+     end
```

Comment:
`end` at 111, 5 is not aligned with `EventStaff.roles.each do |key, value|` at 108, 4.